source/agora/common/Config.d

```diff
@@ -185,7 +185,7 @@ public struct ValidatorConfig
     public string registry_address;
 
     // If the enrollments will be renewed or not at the end of the cycle
-    public bool recurring_enrollment;
+    public bool recurring_enrollment = true;
```

Comment:
This needs to be folded into the previous commit. The previous commit actually says it's off by default, but then the next commit immediately flips it. The default of a feature should be set in the commit which introduces it.